cli/cmd/sync.go

```diff
@@ -35,6 +35,7 @@ func NewCmdSync() *cobra.Command {
 	}
 	cmd.Flags().Bool("no-migrate", false, "Disable auto-migration before sync. By default, sync runs a migration before syncing resources.")
 	cmd.Flags().String("license", "", "set offline license file")
+	cmd.Flags().String("sync-summary-location", "", "Sync summary file location")
```

Comment:
```suggestion
	cmd.Flags().String("summary-location", "", "Sync summary file location")
```

Since we're under the `sync` command I think we can drop the `sync` prefix, WDYT?
Also can we add an example to the `syncExample` `const`?